src/routes/notifications/notifications.controller.spec.ts

```diff
@@ -0,0 +1,243 @@
+import { faker } from '@faker-js/faker';
+import { INestApplication } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import { range, sample } from 'lodash';
+import * as request from 'supertest';
+import {
+  fakeConfigurationService,
+  TestConfigurationModule,
+} from '../../config/__tests__/test.configuration.module';
+import { TestCacheModule } from '../../datasources/cache/__tests__/test.cache.module';
+import { NetworkResponseError } from '../../datasources/network/entities/network.error.entity';
+import {
+  mockNetworkService,
+  TestNetworkModule,
+} from '../../datasources/network/__tests__/test.network.module';
+import { DomainModule } from '../../domain.module';
+import { Chain } from '../../domain/chains/entities/chain.entity';
+import { chainBuilder } from '../../domain/chains/entities/__tests__/chain.builder';
+import { DataSourceErrorFilter } from '../common/filters/data-source-error.filter';
+import { SafeRegistration } from './entities/safe-registration.entity';
+import { registerDeviceDtoBuilder } from './entities/__tests__/register-device.dto.builder';
+import { safeRegistrationBuilder } from './entities/__tests__/safe-registration.builder';
+import { NotificationsModule } from './notifications.module';
+
+describe('Notifications Controller (Unit)', () => {
+  let app: INestApplication;
+
+  const safeConfigUrl = faker.internet.url();
+
+  beforeAll(async () => {
+    fakeConfigurationService.set('safeConfig.baseUri', safeConfigUrl);
+    fakeConfigurationService.set('exchange.baseUri', faker.internet.url());
+    fakeConfigurationService.set(
+      'exchange.apiKey',
+      faker.random.alphaNumeric(),
+    );
+  });
+
+  beforeEach(async () => {
+    jest.clearAllMocks();
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [
+        // feature
+        NotificationsModule,
+        // common
+        DomainModule,
+        TestCacheModule,
+        TestConfigurationModule,
+        TestNetworkModule,
+      ],
+    }).compile();
+
+    app = moduleFixture.createNestApplication();
+    app.useGlobalFilters(new DataSourceErrorFilter());
+
+    await app.init();
+  });
+
+  describe('POST /register/notifications', () => {
+    it('Success', async () => {
+      const chains: Chain[] = range(4).map((idx) =>
```

Comment:
nitpick: `chainId` instead of `idx`

Reply:
Changed in [5bf56a7](https://github.com/5afe/safe-client-gateway-nest/pull/267/commits/5bf56a7f8f6e978eaa8428dc0bf8ac3df335c6cb)